index.js

```diff
@@ -51,13 +53,16 @@ exports.generateFdf = function(data, fileName) {
   return fdf;
 }
 
-exports.generatePdf = function(data, templatePath,callback) {
+exports.generatePdf = function(data, templatePath, callback) {
   var tempName = temp.path({suffix: '.pdf'});
   var tempNameResult = temp.path({suffix: '.pdf'});
 
-  child = spawn("pdftk", ["app/"+templatePath,"fill_form","-","output",tempName,"flatten"]);
+  child = spawn("pdftk", [__dirname+templatePath,"fill_form","-","output",tempName,"flatten"]);
```

Comment:
Or could the implementing code pass is whatever templatePath they'd like instead of being bound to the `__dirname`?